addons/web_editor/static/src/js/editor/transcoder.js

```diff
@@ -229,9 +246,30 @@ function classToStyle($editable) {
         } else {
             $target.attr('style', style);
         }
+        // Apple Mail
         if (this.nodeName === 'TD' && !this.childNodes.length) {
             this.innerHTML = '&nbsp;';
         }
+
+        // Outlook
+        if (this.nodeName === 'A' && $target.hasClass('btn') && !$target.children().length) {
+            var $hack = $('<table class="o_outlook_hack"><tr><td></td></tr></table>');
```

Comment:
I am sure other than outlook have bugs too. I would name this "o_table_btn_hack" (so that we know it is a button too). But do as you wish.